.travis.yml

```diff
@@ -5,7 +5,7 @@ language: generic
 # Alternative to "language: java", which has node installed
 before_install:
   - sudo apt-get update -qq
-  - sudo apt-get install -qq nodejs
+  - sudo apt-get install -qq nodejs maven
```

Comment:
These programs are for use by the Travis process.
If you want to install mvn for use in the Docker container, that needs to be done in `checker/bin-devel/Dockerfile-ubuntu-jdkany` (and the new Docker container built and uploaded).
I can do that for you.